sys/shell/commands/sc_sx127x.c

```diff
@@ -0,0 +1,586 @@
+/*
```

Comment:
out of topic: do you know a way to access the device handler (pointer to `netdev_t` or `sx127x_t`) from `sc_sx127x.c` or other shell commands ? I wanted to provide more CLI features such as access time on air of a packet or device internal temperature which require direct access to driver API whereas I only have access to thread pid there.